packages/api/src/auth.js

```diff
@@ -1,11 +1,12 @@
 import * as JWT from './utils/jwt.js'
```

Comment:
It'd be great to add some documentation, in this folder to add some rationale on how we're handling the different levels of auth, restrictions, psa, banning etc so that we can refer back to it to understand our choices.

Reply:
Done in readme